neural_compressor/transformers/models/modeling_auto.py

```diff
@@ -60,12 +86,13 @@ def build_woq_model(model, quantization_config):
                 "zero_point",
                 not getattr(quantization_config, "sym", False),
             )
-            use_optimum_format = True
+            dtype = "int4" if weight_dtype == "int4_clip" else weight_dtype
```

Comment:
the weight dtype is only int4, int4_fullrange and int8, Could you clean the dtype setting or check?